src/v2/Apps/Artwork/Components/ArtworkSidebar/LotTimer.tsx

```diff
@@ -30,7 +66,9 @@ export const LotTimer: React.FC<LotTimerProps> = ({ saleArtwork }) => {
     hasStarted,
     lotsAreClosing: false,
     isSaleInfo: false,
-    extendedBiddingEndAt,
+    extendedBiddingEndAt: isExtended
```

Comment:
What is the purpose of the ternary operator here?

Reply:
yeah a little confused by this as well

Reply:
`extendedBiddingEndAt` can be in the past when the lot is extended the first time (after you load the page), so the timer label doesn't shift to show `Extended: ...` when that occurs. Reloading the page is fine, the helper needs to know the websocket-updated time for that scenario.

Reply:
ohhhh I see that makes sense!